lib/rubocop/runner.rb

```diff
@@ -27,7 +27,11 @@ def initialize(options, config_store)
 
     def run(paths)
       target_files = find_target_files(paths)
-      inspect_files(target_files)
+      if @options[:list_target_files]
+        puts target_files.map { |file| PathUtil.relative_path(file) }
```

Comment:
Maybe a question of taste, but I would extract this into its own method, and call `puts` inside the loop instead of the implicit dependency on `Array#to_s` for format the output correctly. Something like:
```rb
def output_target_files(files)
  files.each do |file|
    puts PathUtil.relative_path(file)
  end
end
```